doc/nrf/ug_matter_configuring_protocol.rst

```diff
@@ -14,11 +14,15 @@ This page describes what is needed to start working with Matter in the |NCS|.
 Mandatory configuration
 ***********************
 
-To use the Matter protocol, enable the :kconfig:option:`CONFIG_CHIP` Kconfig option.
-Setting this option enables the Matter protocol stack and other associated Kconfig options, including :kconfig:option:`CONFIG_CHIP_ENABLE_DNSSD_SRP` that is required for the discovery of the Matter device using DNS-SD.
+To use the Matter protocol, set the following Kconfig options:
 
-After that, make sure to set the :kconfig:option:`CONFIG_CHIP_PROJECT_CONFIG` Kconfig option and define the path to the configuration file that specifies Vendor ID, Product ID, and other project-specific Matter settings.
+* :kconfig:option:`CONFIG_CHIP` - This option enables the Matter protocol stack and other associated Kconfig options, including :kconfig:option:`CONFIG_CHIP_ENABLE_DNSSD_SRP` that is required for the discovery of the Matter device using DNS-SD.
+* :kconfig:option:`CONFIG_CHIP_PROJECT_CONFIG` - This option defines the path to the configuration file that specifies Vendor ID, Product ID, and other project-specific Matter settings.
 
+Because Matter is an application layer on top of the IPv6-based transport protocols (see :ref:`ug_matter_architecture`), it uses other software modules with their own configuration options, such as Bluetooth® LE, the IPv6 stack (currently, only :ref:`Thread <ug_thread_configuring>` is supported), or :ref:`nRF Security <nrfxlib:nrf_security_config>`.
```

Comment:
That looks pretty good to me, but I would maybe even expand it with more information. Please consider if you find it relevant: 
```suggestion
Because Matter is an application layer protocol on top of the other transport protocols (see :ref:`ug_matter_architecture`), it uses multiple software modules with their own configuration options, to provide communication between the devices and necessary functionalities. It utilizes modules such as Bluetooth® LE, the IPv6 stack (currently, only :ref:`Thread <ug_thread_configuring>` is supported), or :ref:`nRF Security <nrfxlib:nrf_security_config>` and :ref:`MCUboot <mcuboot:mcuboot_ncs>`.
```